### ISSUE TITLE
[Catalog] Internal issue: b/144563222

### ISSUE THREAD

**Issue by material-mirror** [material-components/material-components-ios#8918]
PageControlSwiftExampleViewController has too low contrast on the page control dots. The contrast ratio should be at least 3:1 and is around 1.3:1 on some pages.

This was filed as an internal issue. If you are a Googler, please visit [b/144563222](http://b/144563222) for more details.

---
#### Internal data
- Associated internal bug: [b/144563222](http://b/144563222)
